app/components/autocomplete/autocomplete.tsx

```diff
@@ -102,75 +99,73 @@ const Autocomplete = ({
     const appsTakeOver = showingAppCommand;
     const showCommands = !(showingChannelMention || showingEmoji || showingAtMention);
 
-    const wrapperStyles = useMemo(() => {
-        const s = [];
-        if (Platform.OS === 'ios') {
-            s.push(style.shadow);
-        }
-        return s;
-    }, [style]);
+    const isLandscape = dimensions.width > dimensions.height;
+    const maxHeightAdjust = (isTablet && isLandscape) ? MAX_LIST_TABLET_DIFF : 0;
+    const defaultMaxHeight = MAX_LIST_HEIGHT - maxHeightAdjust;
+    const maxHeight = useDerivedValue(() => {
+        return Math.min(availableSpace.value, defaultMaxHeight);
+    }, [defaultMaxHeight]);
+
+    const containerAnimatedStyle = useAnimatedStyle(() => {
+        return growDown ?
+            {top: position.value, bottom: Platform.OS === 'ios' ? 'auto' : undefined, maxHeight: maxHeight.value} :
+            {top: Platform.OS === 'ios' ? 'auto' : undefined, bottom: position.value, maxHeight: maxHeight.value};
+    }, [growDown]);
```

Comment:
```suggestion
    }, [growDown, position]);
```

Reply:
In general, I would say that this is not needed, since position is a shared value (similar behavior as a reference for dependency checks).

But since it is coming from a prop, we shouldn't make assumptions over that always being the same shared value (i.e. the same reference). Therefore let's add it.